docs/content/en/docs/concepts/architecture/components/scheduler/_index.md

```diff
@@ -6,11 +6,50 @@ weight: 80
 cascade:
 ---
 
-**Keptn's Lifecycle Scheduler** replaces the
-[Kubernetes scheduler](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/)
-to allow users to schedule events and tasks to occur
-at specific times during the application lifecycle.
-The Lifecycle Scheduler can trigger events such as
-deployment, testing, and remediation at specific times or intervals.
-The Keptn Scheduler guarantees that Pods are initiated only after
-the Pre-Deployment checks are completed.
+The **Keptn Scheduler**, an integral component of the Keptn Lifecycle Toolkit which orchestrates
+the deployment process.
+The **Keptn Scheduler** works by registering itself as a Permit plugin with Kubernetes that
+is responsible for ensuring that Pods are scheduled to a node until and unless the
+pre-deployment checks have finished successfully.
+This is important because it helps to prevent Pods from being scheduled to nodes that are
+not yet ready for them, which can lead to errors and downtime
+
+## How does the Keptn Scheduler works
+
+The Keptn Scheduler first checks the annotations on the Pod to see if it is annotated with
```

Comment:
The mutating webhook checks the annotations, and assigns the keptn scheduler only if they are present. So the Keptn scheduler only gets the pods that are annotated.